packages/web-components/src/components/masthead/__stories__/masthead.stories.ts

```diff
@@ -115,46 +92,30 @@ export const Default = (args) => {
     ${useMock
       ? html`
           <dds-masthead-composite
-            platform="${ifDefined(platform)}"
-            .platformUrl="${ifDefined(platformData.url)}"
-            selected-menu-item="${ifDefined(selectedMenuItem)}"
-            user-status="${ifDefined(userStatus)}"
-            searchPlaceholder="${ifDefined(searchPlaceholder)}"
-            .authenticatedProfileItems="${ifDefined(authenticatedProfileItems)}"
+            selected-menu-item="${ifNonNull(selectedMenuItem)}"
```

Comment:
mmissing `ifNonNull` import: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/actions/runs/5556085716/jobs/10148137298?pr=10679

Reply:
good catch!